src/main/features/claimant-response/routes/court-offer.ts

```diff
@@ -0,0 +1,114 @@
+import * as express from 'express'
+
+import { Paths as ClaimantsResponsePaths } from 'claimant-response/paths'
+import { AcceptCourtOffer } from 'claimant-response/form/models/acceptCourtOffer'
+import { Form } from 'forms/form'
+import { ErrorHandling } from 'shared/errorHandling'
+import { FormValidator } from 'forms/validation/formValidator'
+import { YesNoOption } from 'models/yesNoOption'
+
+import { Claim } from 'claims/models/claim'
+import { PartialAdmissionResponse } from 'claims/models/response/partialAdmissionResponse'
+import { FullAdmissionResponse } from 'claims/models/response/fullAdmissionResponse'
+
+import { DraftClaimantResponse } from 'claimant-response/draft/draftClaimantResponse'
+import { Draft } from '@hmcts/draft-store-client'
+import { DraftService } from 'services/draftService'
+
+import { PaymentPlan } from 'common/payment-plan/paymentPlan'
+import { Frequency } from 'common/frequency/frequency'
+import { PaymentPlanHelper } from 'shared/helpers/paymentPlanHelper'
+import { CourtOrderHelper } from 'shared/helpers/courtOrderHelper'
+
+import { IncomeExpenseSource } from 'common/calculate-monthly-income-expense/incomeExpenseSource'
+import { CalculateMonthlyIncomeExpense } from 'common/calculate-monthly-income-expense/calculateMonthlyIncomeExpense'
+import { Expense } from 'claims/models/response/statement-of-means/expense'
+import { Income } from 'claims/models/response/statement-of-means/income'
+
+function calculateTotalMonthlyIncome (incomes: Income[]): number {
+  if (incomes === undefined) {
```

Comment:
is it undefined by default rather than an empty array?

Reply:
its undefined by default